adapters/bizzclick/bizzclick.go

```diff
@@ -104,7 +104,11 @@ func (a *adapter) getImpressionExt(imp *openrtb2.Imp) (*openrtb_ext.ExtBizzclick
 }
 
 func (a *adapter) buildEndpointURL(params *openrtb_ext.ExtBizzclick) (string, error) {
-	endpointParams := macros.EndpointTemplateParams{AccountID: params.AccountID, SourceId: params.PlacementID}
+	host := "us-e-node1"
+	if params.Host != "" {
+		host = params.Host
+	}
```

Comment:
should add/ update test where `params.Host` is empty

Reply:
The test was added here:
`adapters/bizzclick/bizzclicktest/exemplary/default-host-param.json`
With the placementId param also